.github/workflows/build_and_release_gem.yml

```diff
@@ -76,28 +76,27 @@ jobs:
           exit 1
 
       - name: Create release draft that includes the checksum
-        uses: actions/github-script@v3
-        with:
-          github-token: ${{secrets.GITHUB_TOKEN}}
-          script: |
-            github.repos.createRelease({
-              owner: "solarwinds-apm",
-              repo: "appoptics-apm-ruby",
-              body: "SHA256 checksum: ${{ steps.build.outputs.checksum }}",
-              tag_name: "${{ steps.build.outputs.gem_version }}",
-              name: "${{ steps.build.outputs.gem_version }}",
-              draft: true
-            })
-
-      # may need a bit of time for the gem to become available (sleep 1)
+        run: |
+          curl \
+             -X POST \
+             -H "Accept: application/vnd.github.v3+json" \
+             https://api.github.com/repos/OWNER/REPO/releases \
+             -d '{"tag_name":"${{ steps.build.outputs.gem_version }}","name":"${{ steps.build.outputs.gem_version }}","body":"SHA256 checksum: ${{ steps.build.outputs.checksum }}","draft":true}'
```

Comment:
AFAICT `actions/github-script` is still maintained by GHA, so wondering why this change?  Also, how are credentials being given by curl when making the call to GH API, and are `OWNER/REPO` getting set correctly?

Reply:
the create_release script is not maintained anymore and not working anymore
When I wrote the release notes I realized that this version is not working either, but it doesn't fail the action 
Let me see if I can fix it.